src/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+
+const addBtn = document.querySelector('#add-btn');
+const todoList = document.querySelector('.todo-list');
+const inputField = document.querySelector('#input-field');
+const clearAll = document.querySelector('.clearAll');
+
+const todoListArray = [];
+
+const addnew = () => {
+  todoListArray.push(inputField.value);
```

Comment:
- [ ] Your to-do list should be an array that contains objects with fields, description [`string`], completed [`bool`], and index [`number`]. It seems your to-do items are only a `string`, please make sure to add the necessary fields.

```javascript
// Example of a to-do array with items
const todoListArray = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```